### PR TITLE
docs: LSDV-5358: Add Generative AI templates

### PR DESCRIPTION
**Pull request by niklub**
### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: docs/source/templates/generative-supervised-llm.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+type: templates
+category: Generative AI
+cat: generative-ai
+order: 903
+is_new: t
+meta_title: Create dataset for supervised LLM fine-tuning
+meta_description: Template for creating dataset for supervised LLM fine-tuning with Label Studio for your machine learning and data science projects.
+---
+
+The goal of Supervised LLM Fine-tuning is to optimize your LLM to generate responses given user-defined prompts. There
```
Comment (hogepodge) [inline docs/source/templates/generative-supervised-llm.md]: Currently, line breaks in markdown are rendered as line breaks in the generated text. If that is the desired outcome of our rendering engine (which is [out of spec with the markdown standard](https://daringfireball.net/projects/markdown/syntax#precode)), then all paragraphs in the document should be on one line.

**Comment by erinmikailstaples** [comment]
Updated items in the latest commit — I didn't catch the spacing error that @hogepodge did on preview, but I mostly cleaned up grammar, added links, and corrected typos in the updates made. 

LGTM! 🚀 

@niklub — ready when you are

File: docs/source/templates/get_started.md
Comment (erinmikailstaples) [inline docs/source/templates/get_started.md:1]: FYI — added more context and detail here @niklub to hone in on specificity and hopefully make our tags easier to find.

**Review by erinmikailstaples — APPROVED** [review]
Reviewed the formatting in preview and with @huguestennier's adjustments going to give it another review here shortly to make sure the docs are all consistent. 

@niklub LGTM otherwise 🚀

**Review by erinmikailstaples — APPROVED** [review]

**Pull request merged**